.github/workflows/build.yml

```diff
@@ -0,0 +1,32 @@
+name: build
+
+env:
+  VERSION: ${{ format('{0}.{1}.{2}', 0, 2, github.run_number) }}
```

Comment:
will this override the version set for hdlcpp in [CMakeLists.txt:3](https://github.com/bang-olufsen/hdlcpp/pull/30/files#diff-1e7de1ae2d059d21e1dd75d5812d5a34b0222cef273b7c3a2af62eb747f9d20aR3)

Reply:
No, but we could. I added the env just in case we want to use it. Currently we don't have any release triggers - can remove for now if it's confusing :+1: